src/python/grpcio_tests/tests/unit/framework/interfaces/face/_future_invocation_asynchronous_event_service.py

```diff
@@ -450,6 +452,7 @@ def testFailedUnaryRequestUnaryResponse(self):
           with self.assertRaises(face.ExpirationError):
             response_future.result()
           self.assertIsNotNone(response_future.traceback())
+          self.assertTrue(abortion_callback._called)
```

Comment:
Oh, and since `_Callback.future` returns the first positional argument that was passed to the callback or `None` if zero positional arguments were passed, I suppose my recommendation here changes to `self.assertIsNone(abortion_callback.future())`.